### DISCUSSION TITLE
Map in one serie

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1917] [Q&A] [answered]
Hello

If I have two series in the same data file, can a label be set to show a map in each of them?

Thank you

**Comment by sarah-beasley** [comment, marked as answer]
Hi, it is possible to have one map with multiple series, you have to toggle between the series. This can be done by enabling the 'disaggregation_controls: true' under your map options - https://open-sdg.readthedocs.io/en/latest/maps/ . This will create a button which will let you choose the series you want to display on the map and change between them.
